src/ui/src/js/run.js

```diff
@@ -151,13 +148,39 @@ export default function appRun(
     // );
   };
 
-  $rootScope.hasPermission = function(user, permissions) {
+  $rootScope.hasPermission = function(user, permission) {
```

Comment:
There's a way to do this that doesn't involve adding a bunch of stuff to the `rootScope`, but since that's largely the existing pattern I just stuck with it here, since it does make the code as it currently exists easier to work with.